README.md

```diff
@@ -47,29 +47,64 @@ Install the chart using the name of the plugin as the Helm release name into a n
 helm upgrade -i troubleshooting-panel-console-plugin charts/openshift-console-plugin -n troubleshooting-panel-console-plugin --create-namespace --set plugin.image=quay.io/my-repository/troubleshooting-panel-console-plugin:latest
 ```
 
-
 ## Development
 
 ### Dependencies 
 1. [yarn](https://yarnpkg.com/en/docs/install)
 2. [oc](https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.4/) 
 3. [podman 3.2.0+](https://podman.io) or [Docker](https://www.docker.com) 
 4. An OpenShift cluster
+5. [Korrel8r](https://korrel8r.github.io/korrel8r) instance running in the cluster. Assumed to be running v0.0.8
 
+>[!WARNING]
+> The Korrel8r instructions state that the default instance should be in the namespace `korrel8r`
+> With the name `korrel8r-instance`. However, the logging plugin expects the name to be `korrel8r` 
+> and we are following the same convention so that the plugins can be run together.
 
+#### TP (Troubleshooting Panel) Development Server
 In one terminal window, run:
+1. `yarn install`
```

Comment:
From #3 we are using npm instead of yarn

Reply:
Updated all instances of yarn in the readme to their npm command counterparts